### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by m7pr**
Related to https://github.com/insightsengineering/coredev-tasks/issues/462

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,17 +34,31 @@ This package provides various 'widgets' for `teal` applications. Examples includ
...
+install.packages('teal.widgets', repos = c('https://pharmaverse.r-universe.dev', getOption('repos')))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal.widgets")
 ```
 
-See package vignettes `browseVignettes(package = "teal.widgets")` for usage of this package.
+## Usage
+
+To understand how to use this package, please refer to the [package website](https://insightsengineering.github.io/teal.widgets/), which provides multiple examples of code usage.
```
Comment (donyunardi) [inline README.md:57]:
```suggestion
To understand how to use this package, please refer to the [package website](https://insightsengineering.github.io/teal.widgets/latest-tag/), which provides multiple examples of code usage.
```

Comment (donyunardi) [reply README.md:57]: Also, just like what we did with teal.code and teal.logger, is it possible to add a few line of code to showcase the usage of teal.widgets?

**Review by donyunardi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (m7pr) [reply README.md:57]: Hey @donyunardi `https://insightsengineering.github.io/teal.widgets/` automatically redirects to `https://insightsengineering.github.io/teal.widgets/latest-tag/` so no need for explicit usage of `latest-tag/` in the name. We put `latest-tag/` in urls that contains anything after `teal.widgets/` like `https://insightsengineering.github.io/teal.widgets/index.html` or ``https://insightsengineering.github.io/teal.widgets/articles/index.html` so it's `https://insightsengineering.github.io/teal.widgets/latest-tag/index.html` or ``https://insightsengineering.github.io/teal.widgets/latest-tag/articles/index.html`. But no need for regular website that ends with `teal.widgets`.

Comment (m7pr) [reply README.md:57]: > Also, just like what we did with teal.code and teal.logger, is it possible to add a few line of code to showcase the usage of teal.widgets?

@donyunardi the code for teal.widgets is way too big for quick view on README and I decided we will only point to the website for this package. I doesn't have any quick short snippets. It rather have massive code.

**Comment by kartikeyakirar** [comment]
GitHub Pages automatically searches for an index.html file, so is it necessary to include "index.html" at the end of the URL? I was just was curious 🤔

**Comment by m7pr** [comment]
No it's not! But let's leave it as it is for now

File: README.md (hunk 1 above)
Comment (donyunardi) [reply README.md:57]: Okay, that's fine.

**Review by donyunardi — APPROVED** [review]
LGTM

**Pull request merged**